### PR TITLE
DBZ-1767 Support field sanitization for MongoDB unwrap

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-1767

### CODE REVIEW

**Comment by jpechane** [comment]
@gunnarmorling The one failure is duu to log size

**Comment by gunnarmorling** [comment]
Can you rebase and force-push?

>

File: documentation/modules/ROOT/pages/configuration/mongodb-event-flattening.adoc [hunk 1]
```diff
@@ -279,6 +279,11 @@ For `DELETE` events, this option is only supported when the `delete.handling.mod
 |`add.source.fields`
 |
 |Fields from the change event's `source` structure to add as metadata (prefixed with "__") to the flattened record
+
+|`sanitize.field.names`
+|`false`
+|Whether field names will be sanitized to adhere to Avro naming requirements.
+See xref:https://debezium.io/documentation/reference/1.0/configuration/avro.html#names[Avro naming] for more details.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/configuration/mongodb-event-flattening.adoc]:
```suggestion
See xref:configuration/avro.adoc#names[Avro naming] for more details.
```

Let's avoid absolute links like that, in particular as we want this to point to the same version of the docs as the document at hand.

**Comment by gunnarmorling** [comment]
Unrelated test failure is taken care of in parallel PR #1257.

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks, @jpechane!

**Pull request closed without merging**